client/src/components/JurisdictionAdmin/RoundManagement.test.tsx

```diff
@@ -234,31 +226,18 @@ describe('RoundManagement', () => {
       const downloadBatchTallySheetsButton = screen.getByRole('button', {
         name: /Download Batch Tally Sheets/,
       })
-      expect(downloadBatchTallySheetsButton).toHaveAttribute(
-        'href',
-        'blob:http://blob-url'
-      )
-
-      // Expect a @react-pdf/renderer Page per batch (note that a Page can span multiple actual
-      // pages). Page contents are further tested in BatchTallySheet.test.tsx
-      expect(Page).toHaveBeenCalledTimes(3)
-      const pdf = screen.getByTestId('pdf')
-      expect(
-        within(pdf).queryAllByText('Audit Board Batch Tally Sheet')
-      ).toHaveLength(3)
-      await within(pdf).findByText('Batch One')
-      await within(pdf).findByText('Batch Two')
-      await within(pdf).findByText('Batch Three')
-      expect(within(pdf).queryAllByText('Candidates/Choices')).toHaveLength(3)
-      expect(within(pdf).queryAllByText('Enter Stack Totals')).toHaveLength(3)
-      expect(within(pdf).queryAllByText('Choice One')).toHaveLength(3)
-      expect(within(pdf).queryAllByText('Choice Two')).toHaveLength(3)
-
       screen.getByRole('table') // Tested in BatchRoundDataEntry.test.tsx
+
+      act(() => {
+        userEvent.click(downloadBatchTallySheetsButton)
+      })
+      // PDF contents are tested in BatchTallySheet.test.tsx
```

Comment:
Still trying to see if I can do some rendering in this test as I was earlier - haven't been able to get things working quite yet

Reply:
Sorted this out! https://github.com/votingworks/arlo/pull/1475/commits/4981c6821a6c90a55f3a8fb66d5ce7501e96a6bd